Testing/functional/tests/cypress/integration/e2e/timeline/mobile/clinicaldocument.js

```diff
@@ -0,0 +1,31 @@
+const { AuthMethod } = require("../../../../support/constants");
+
+describe("Clinical Document", () => {
+    beforeEach(() => {
+        cy.enableModules("ClinicalDocument");
+        cy.viewport("iphone-6");
+        cy.login(
+            Cypress.env("keycloak.username"),
+            Cypress.env("keycloak.password"),
+            AuthMethod.KeyCloak
+        );
+        cy.checkTimelineHasLoaded();
+    });
+
+    it("Validate Card Details for Mobile", () => {
+        cy.get("[data-testid=timelineCard]").last().scrollIntoView().click();
+
+        const entryDetails = cy.get("[data-testid=entryDetailsCard]");
```

Comment:
This pattern is not supported by cypress.  There are several options that may be a suitable alternative, depending on the context:

- chaining .within()
- chaining .then()
- storing an alias using .as()

[This page](https://docs.cypress.io/guides/core-concepts/variables-and-aliases) has some additional information.

Reply:
I think .within() is the appropriate choice here.

Reply:
Changed to within.